src/plugins/flexibleLayout/toolbarProvider.js

```diff
@@ -0,0 +1,167 @@
+export default {
```

Comment:
This needs to be a at minimum a function that returns an object, preferably a class that you construct -- you can't expect that openmct will be available in global state; it should be passed in.